javascript-practice/src/js/constants/message.js

```diff
@@ -1,12 +1,12 @@
-const Validation_Errors = {
+const VALIDATE_ERORR = {
```

Comment:
Required_Field not match with format in REGEX

Reply:
I done Fix it